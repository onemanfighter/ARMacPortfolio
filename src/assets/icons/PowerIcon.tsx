import * as React from 'react';

const PowerIcon = (props: React.SVGProps<SVGSVGElement>) => {
  return (
    <svg
      fill="currentColor"
      viewBox="0 0 16 16"
      height="1.5em"
      width="1.5em"
      {...props}
    >
      <path d="M7.5 1v7h1V1h-1z" />
      <path d="M3 8.812a4.999 4.999 0 012.578-4.375l-.485-.874A6 6 0 1011 3.616l-.501.865A5 5 0 113 8.812z" />
    </svg>
  );
};

export { PowerIcon };
