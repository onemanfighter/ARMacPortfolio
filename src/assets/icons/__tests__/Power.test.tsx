import { render } from '@testing-library/react';
import { PowerIcon } from '../PowerIcon';

describe('PowerIcon', () => {
  it('should render correctly', () => {
    const { container } = render(<PowerIcon />);

    expect(container).toMatchSnapshot();
  });
});
