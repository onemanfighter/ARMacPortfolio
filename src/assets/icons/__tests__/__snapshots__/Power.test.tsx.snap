// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PowerIcon should render correctly 1`] = `
<div>
  <svg
    fill="currentColor"
    height="1.5em"
    viewBox="0 0 16 16"
    width="1.5em"
  >
    <path
      d="M7.5 1v7h1V1h-1z"
    />
    <path
      d="M3 8.812a4.999 4.999 0 012.578-4.375l-.485-.874A6 6 0 1011 3.616l-.501.865A5 5 0 113 8.812z"
    />
  </svg>
</div>
`;
